.github/workflows/resetTrainingDB.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy Training
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run this on a daily timer as well in order to reset the training DB
+    - cron: '* 9 * * *'
+
+env:
+  DEPLOY_ENV: training
+  NODE_VERSION: 14
+concurrency:
+  group: training-deploy
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./backend
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to ACR
+        run: docker login ${{ secrets.ACR_REPO_URL }} -u ${{ secrets.ACR_ADMIN_USERNAME }} -p ${{ secrets.ACR_ADMIN_PASWORD }}
+      - name: Build and push Docker images
+        run: ./build_and_push.sh
+  deploy-backend:
+    runs-on: ubuntu-latest
+    needs: docker-build
+    defaults:
+      run:
+        working-directory: ./ops
+    env:
+      ARM_CLIENT_ID: ${{ secrets.TERRAFORM_ARM_CLIENT_ID }}
+      ARM_CLIENT_SECRET: ${{ secrets.TERRAFORM_ARM_CLIENT_SECRET }}
+      ARM_SUBSCRIPTION_ID: ${{ secrets.TERRAFORM_ARM_SUBSCRIPTION_ID }}
+      ARM_TENANT_ID: ${{ secrets.TERRAFORM_ARM_TENANT_ID }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.15.1
+      - name: Terraform Init
+        run: make init-${{ env.DEPLOY_ENV }}
+      - name: Replace Azure training DB
+        run: |
+          cd ${{ env.DEPLOY_ENV }}/persistent
```

Comment:
You can assign the working directory to the step rather than doing a chdir here.